lib/src/chart/base/base_chart/render_base_chart.dart

```diff
@@ -50,22 +52,20 @@ abstract class RenderBaseChart<R extends BaseTouchResponse> extends RenderBox
 
   /// Initializes our recognizers and implement their callbacks.
   void initGestureRecognizers() {
-    _panGestureRecognizer = PanGestureRecognizer();
-    _panGestureRecognizer
-      ..onDown = (dragDownDetails) {
-        _notifyTouchEvent(FlPanDownEvent(dragDownDetails));
+    _scaleGestureRecognizer = ScaleGestureRecognizer();
+    _scaleGestureRecognizer
+      ..onStart = (details) {
+        _notifyScaleEvent(FlScaleStartEvent(details));
+        _notifyTouchEvent(FlPanDownEvent(details));
```

Comment:
Is it okay to call both FlPanDown and FlPanStart at the same time?

Reply:
Probably not, drag down details and scalestart details will be different and will be breaking a change. Thinking of another solution